Source Code/Ticket Management System/user.c

```diff
@@ -222,10 +222,10 @@ void user_setPhone(struct User* user, wchar_t* phone)
 				user->contact_details.phone = old_phone;
 				return;
 			}
-			else if (old_phone != user->contact_details.phone)
+			/*else if (old_phone != user->contact_details.phone)
```

Comment:
Why?

Reply:
told you already about it not working, I left it as a comment on purpose.